.azure/pipelines/jobs/iisintegration-job.yml

```diff
@@ -3,8 +3,7 @@ jobs:
   parameters:
     buildScript: ./build.cmd
     buildArgs: "-ci -restore -build -test -projects src/Servers/IIS/**/*.csproj ${{ parameters.skipArgs }}"
-    poolName: "Hosted VS2017"
-    poolVmImage: "vs2017-win2016"
+    poolName: dotnet-external-temp-vs2019
```

Comment:
@pakrym @jkotalik - what would you like to do here? There is no Microsoft-hosted pool that has VS 2019, and the agent in dotnet-external-temp-vs2019 doesn't run with admin rights.

cc @muratg

Reply:
Should we check with Arcade folks for their recommendation?

Is this agent pool something we created? If so, we can change it to run with admin perhaps?
Alternatively, we can clone the pool and change it and use it, right?

Reply:
What are the options? I would prefer the tests running or at least to have a plan and a timeline to reenable them.

Reply:
Options:

* Build tests in VS2019, copy bits over to a different machine for test execution (via Helix or via an 'artifact dependency' in our YML pipeline)
* Create a new different agent pool that has both VS2019 and the admin rights these tests need.
* Don't upgrade to VS2019 yet (but this is going to hold up other things we want to do, like use C# 8)

I like the first option.

Reply:
Option 1 seems reasonable and if moving us in the right direction overall. Altough I don't think IIS tests are in the shape to be published and ran without requiring SDK on other machine.

Reply:
Ok. Let's play around with some approaches. I expect this PR won't go in for a few days while we work on this. We're not exactly in a rush since we're still stabilizing Preview 2, so we have time to find something that works.

Reply:
@pakrym is taking the first steps towards Helix testing in https://github.com/aspnet/AspNetCore/pull/7586. We'll leave disabled the IIS tests which require admin until they are ported to run in Helix.